svg-dealing.md

```diff
@@ -0,0 +1,162 @@
+
+### Hier mal ein paar Beispiele:
+
+`fill="..."` beschreibt die Farbe. Hier kann man Farben (weiter unten im Text) einfügen
+```SVG
+<rect id = "color"
+	x="2" y="2" width="300" height="93" fill="#d01f26"
+/>
+```
+
+*Weimarer Strasse* wird nun als einzeiliger Straßenname angezeigt.
+Bei kurzen Namen reicht das völlig aus.
+```SVG
+<text id = "oneliner"
+	x="150" y="195" font-size="30px" fill="##231f20"
+	style="stroke: #000000" text-anchor="middle">
+	Weimarer Strasse
+</text>
+```
+
+*Weimarer Straße* wird nun als zweizeiliger Straßenname angezeigt.
+Hilfreich bei längeren Namen.
+```SVG
+<text id = "twoliner"
+	font-size="30px" fill="##231f20"
+	style="stroke: #000000" text-anchor="middle">
+	<tspan x="150" y="182">
+		Weimarer
+	</tspan>
+	<tspan x="150" y="212">
+		Stra&#223;e
+	</tspan>
+</text>
+```
+
+Das Grundstück kostet hier *7200 RM*. Der Preis wird immer folgendermaßen angegeben:
+`RM X`, wobei `X` der Preis ist.
+```SVG
+<text id = "price"
+	x="150" y="480" font-family="Monopoly" font-size="33px" text-anchor="middle">
+	RM 7200
+</text>
+```
+
+Verwendete Farben und hilfreiche HTML-Codes
+-------------------------------------------
+
+Bei den Straßennamen ist es sehr wichtig, die Umlaute zu ersetzen, da nicht jedes
+Programm Umlaute anzeigen kann. Gimp, zum Beispiel versagt hier; Firefox versagt
+schon bei der ``transform``-Methode.
+(Fachschaft Kunst approoved.)
+
+Allgemeine Farben |Code           |
+------------------|---------------|
+Schrift, Outlines | ```#231f20``` |
+Hintergrundfarbe  | ```#f7f7cf``` |
+Schwarz           | ```#000000``` |
+
+Eckfelder        |Code           |
+-----------------|---------------|
+Gefängnis-Orange | ```#ca4a24``` |
+Polizist-Blau    | ```#428db7``` |
+Los-Pfeil-Rot    | ```#ed1c24``` |
+Frei-Parken-Rot  | ```#ee2d2b``` |
+
+Straßen      |Code           |
+-------------|---------------|
+Dunkles Lila | ```#30041c``` |
+Helles Blau  | ```#91c3d8``` |
+Helles Lila  | ```#860459``` |
+Orange       | ```#de5126``` |
+Rot          | ```#d01f26``` |
+Gelb         | ```#fbe821``` |
+Grün         | ```#168140``` |
+Dunkles Blau | ```#183a66``` |
+
+Umlaute |Code          |
+--------|--------------|
+Ä       | ```&#196;``` |
+ä       | ```&#228;``` |
+Ö       | ```&#214;``` |
+ö       | ```&#246;``` |
+Ü       | ```&#220;``` |
+ü       | ```&#252;``` |
+ß       | ```&#223;``` |
+§       | ```&#167;``` |
+®       | ```&#174;``` |
+©       | ```&#169;``` |
+
+Straßennamen
+------------
+
+Hier eine Liste der Straßennamen mit zugehörigen Farb-Codes und zusätzlich in "SVG-Sprache" (ohne Umlaute):
+
+Straßenname              |SVG-Straßenname          |Kaufpreis |ohne Haus |1 Haus |2 Häuser |3 Häuser  |4 Häuser |Hotel  |Preis für ein Haus  |Farbe      |SVG-Color     |
+-------------------------|-------------------------|----------|----------|-------|---------|----------|---------|-------|--------------------|-----------|--------------|
```

Comment:
Problem: we need the original `plots.md` because we have to autogenerate code from it.